humble/distribution.yaml

```diff
@@ -2559,6 +2559,13 @@ repositories:
       url: https://github.com/swri-robotics/mapviz.git
       version: ros2-devel
     status: developed
+  marker_msgs:
+    release:
```

Comment:
Would you like to add a `doc` and a `source` section here?  That was done for all ROS 1 releases, e.g. https://github.com/ros/rosdistro/blob/57b8bfd2d425aeba494a33b3834af10e9dbefc21/noetic/distribution.yaml#L4948-L4962